### PR TITLE
85662 Link moved to the right

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description

List of proposed changes:
- added css to keep the link to the right

### What to test for/How to test
- Download and verify

### CODE REVIEW

**Review by alex-solo — DISMISSED** [review]
The "Edit" link looks good but the content changes are not implemented here (take a look at the text changes for short question text in Andreanne's description inside the task in ADO. for example: "Residence country" should be "Country of residence")

**Review by alexunique0519 — DISMISSED** [review]
Screenshot 2022-11-21 at 3 41 01 PM
**The "Edit" link was pushed to the next row in Marital Status section**

**Review by alex-solo — DISMISSED** [review]
The "What you told us" section still looks different from the task description in ADO

**Comment by MarcoGoC** [comment]
> The "What you told us" section still looks different from the task description in ADO

Not sure what you mean. the link has been mentioned before that they want it to the right

File: i18n/api/en.ts [hunk 1]
```diff
@@ -46,7 +46,8 @@ const en: Translations = {
       'When would you like to start receiving the Old Age Security (OAS) pension?',
     [FieldKey.OAS_AGE]:
       'At what age would you like to start receiving the OAS pension?',
-    [FieldKey.MARITAL_STATUS]: 'What is your marital status?',
+    [FieldKey.MARITAL_STATUS]: 'What is your current marital status?',
```
Comment (alexunique0519) [inline i18n/api/en.ts]: Could you remove the "current"? I was asked to remove the "current" in another task. Otherwise, it looks good to me.

**Review by alexunique0519 — APPROVED** [review]
Thanks!

**Pull request merged**